### PR TITLE
Create team page

### LINKED ISSUE #5497: ✨ (Admin) Create new team
## ✨ Feature

This page allows users with team CRUD permissions to create a new team

## 🕵️ Details

Don't worry about the breadcrumbs, which will be handled in #5472 

## 🎨 Design File

[Link to Figma](https://www.figma.com/file/UsrB15yicgDCLDQmx3snyl/User-Flows?node-id=0%3A1&t=HjevjsYNnRyAt9Nr-0)

## 📸 Screenshot

image

## 🙋‍♀️ Proposed Implementation

Allow for editing of the Key (name) field, but populate it with a kebab-case version of the English name on the English display name onblur event.

## 🌎 Localization

(optional) Provide any new copy along with translations available.

## ✅ Acceptance Criteria

A set of assumptions which, when tested, verify that the feature was properly implemented.

- [ ] Admin users can use this page to create a new Team
- [ ] Storybook story for the new page
- [ ] Page actually submits createTeam mutation
- [ ] If creation is successful, redirects user to `/teams/{id}`

### PR DESCRIPTION
**Pull request by JamesHuf**
🤖 Resolves #5497

## 👋 Introduction

Adds a form to the `/admin/teams/create` page.

## 🕵️ Details

Builds off #5499 and attempts limit code duplication between the two forms.

## 🧪 Testing

1. Build the app `npm run build`
2. Navigate to `/admin/teams/create`
3. Fill out the form fields
4. Click the "Create new team" button
5. Confirm the team is created
6. Find your team in the table and click the pencil icon
7. Confirm the fields contain the correct info
8. Add some descriptions
9. Click the "Save team information" button
10. Confirm the team is updated

## 📸 Screenshot

image

### CODE REVIEW

**Comment by brindasasi** [comment]
team voted to remove this field from the UI 
cc: @GCTC-NTGC/designers  , @GCTC-NTGC/developers

File: apps/web/src/pages/Teams/UpdateTeamPage/components/UpdateTeamForm.tsx [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+            label={intl.formatMessage({
+              defaultMessage: "Organization's short description (English)",
+              id: "TCQhlo",
+              description: "Label for the English team description input",
+            })}
+            rules={{
+              required: intl.formatMessage(errorMessages.required),
+            }}
+          />
+          <DescriptionWordCounter name="description.fr" />
```
Comment (brindasasi) [inline apps/web/src/pages/Teams/UpdateTeamPage/components/UpdateTeamForm.tsx:123]: I think the problem is here. It should be "description.en"

Comment (JamesHuf) [reply apps/web/src/pages/Teams/UpdateTeamPage/components/UpdateTeamForm.tsx]: Looks like that was already addressed with [this commit](https://github.com/GCTC-NTGC/gc-digital-talent/pull/5743/commits/c929e251da7d646c55f8e962b233f2de85aa8f7f) which has now been merged in.

**Review by brindasasi — APPROVED** [review]
Works great! Good to go! 🥳

**Pull request merged**